File: 3658-gcd-of-odd-and-even-sums/3658-gcd-of-odd-and-even-sums.cpp
class Solution {
public:
    int gcdOfOddEvenSums(int n) {
        int odd= 0 , even = 0;
        for (int i =1 ; i<=n*2 ;i++){
            if(i%2==0){
                even+=i;
            }
            else{
                odd+=i;
            }
        }
        return gcd(odd,even);
        
    }
};

File: 3658-gcd-of-odd-and-even-sums/3658-gcd-of-odd-and-even-sums.py
class Solution:
    def gcdOfOddEvenSums(self, n: int) -> int:
        oddsum=0
        evensum=0
        for i in range (2*n):
            if(i%2==0):
                evensum+=i
            else:
                oddsum+=i
        return gcd(evensum,oddsum)
        